packages/ext-react-boilerplate/ext-react/workspace.json

```diff
@@ -1,7 +1,7 @@
 {
     "apps": [],
     "frameworks": {
-        "ext": "../node_modules/@sencha/ext-modern"
+        "ext": "../node_modules/@sencha/ext"
```

Comment:
Not sure if this has any other impact, but `ext-modern` package is a `toolkit` package, not a `framework` package, and Cmd therefore threw an error.